tracing-subscriber/src/fmt/mod.rs

```diff
@@ -296,9 +296,13 @@ use crate::subscribe::Subscribe as _;
 use crate::{
     filter::LevelFilter,
     registry::{LookupSpan, Registry},
-    reload, subscribe,
+    subscribe,
 };
 
+#[cfg(feature = "reload")]
```

Comment:
Could you use the `cfg_feature!` macro (https://github.com/tokio-rs/tracing/blob/master/tracing-subscriber/src/lib.rs#L128) instead? Would be a bit more consistent here.